src/app/work-item/work-item.component.html

```diff
@@ -1,83 +1,99 @@
-        <div class="wiTypeListBlk">
-          <i almIcon [iconType]="type.name" class="xl-font"></i>
-          <p>{{type.name.substring(7, type.length)}}</p>
-        </div>
-      </li>
-    </ul>
-  </div>
-
-  <div class="work-item-action-bar">
-    <div class="list-action">
-
-      <!--Move a work item up / down -->
-      <div class="dropdown move-dropdown" dropdown>
-        <button class="btn btn-default dropdown-toggle" type="button" id="wi_filter_dropdown" data-toggle="dropdown" dropdown-open>
-          Move
-          <span class="caret"></span>
-        </button>
-        <ul class="dropdown-menu dropdown-menu-left" role="menu" aria-labelledby="wi_filter_dropdown">
-          <li>
-            <a role="menuitem" tabindex="-1" (click)="moveItem('top')">Move to Top</a>
-          </li>
-          <li>
-            <a role="menuitem" tabindex="-1" (click)="moveItem('bottom')">Move to Bottom</a>
-          </li>
-          <li role="presentation" class="divider"></li>
-          <li>
-            <a role="menuitem" tabindex="-1" (click)="moveItem('up')">Move Up</a>
-          </li>
-          <li>
-            <a role="menuitem" tabindex="-1" (click)="moveItem('down')">Move Down</a>
-          </li>
-        </ul>
-      </div>
-
-      <!--Filter-->
-      <div class="dropdown filter-dropdown" dropdown>
-        <button class="btn btn-default dropdown-toggle" type="button" id="wi_filter_dropdown" data-toggle="dropdown" dropdown-open>
-          Filter Method
-          <span class="caret"></span>
-        </button>
-        <ul class="dropdown-menu dropdown-menu-left" role="menu" aria-labelledby="wi_filter_dropdown">
-          <li *ngIf="!filters.length">
-            <span>No filters available</span>
-          </li>
-          <li *ngFor="let filter of filters;" role="presentation" (click)="activeFilter(filter.id)">
-            <a role="menuitem" tabindex="-1">{{filter.name}}</a>
-          </li>
-        </ul>
+    <div class="container-fluid container-cards-pf">
+      <div class="row row-cards-pf">
+      <div class="col-md-12 col-sm-12 col-xs-12 no-padding">
+        <h2 class="pull-left">Choose a new work item type </h2>
+        <i class="pficon-close detail-close pull-right" (click)="closePanel()"></i>
       </div>
-
-      <div *ngIf="loggedIn" class="add-detailed-wi"
-          tooltipPlacement="bottom" tooltip="Add A Work Item"
-          (click)="showTypes()">
-        <span class="fa fa-plus icon"></span>
+        <div *ngFor="let type of workItemTypes" class="col-xs-12 col-sm-6 col-md-4 col-lg-3" (click)="onChangeType(type.name)">
+          <div class="card-pf card-pf-view card-pf-view-select">
+          <div class="card-pf-body">
+            <div class="card-pf-top-element">
+              <i almIcon [iconType]="type.name" class="card-pf-icon-circle"></i>
+            </div>
+            <h2 class="card-pf-title text-center">
+              {{type.name.split(".")[1]}}
```

Comment:
Need to change this to {{ type.name }}.  Now you can get WI type directly by type.name.

https://github.com/fabric8io/fabric8-planner/pull/782